scripts/quests/bastok/Return_to_the_Depths.lua

```diff
@@ -0,0 +1,256 @@
+                        return quest:progressEvent(40, 0, xi.items.SPRIG_OF_FRESH_MUGWORT, xi.items.MOORISH_IDOL, xi.items.PIECE_OF_ANGEL_SKIN, xi.items.BULB_OF_MISAREAUX_GARLIC)
+                    elseif quest:getVar(player, 'prog') == 5 then
+                        return quest:progressEvent(41)
+                    end
+                end,
+            },
+
+            onEventFinish =
+            {
+                [40] = function(player, csid, option, npc)
+                    quest:setVar(player, 'prog', 1)
+                end,
+
+                [41] = function(player, csid, option, npc)
+                    quest:setVar(player, 'prog', 6)
+                end,
+
+                [42] = function(player, csid, option, npc)
+                    if option == 0 then
+                        quest:setVar(player, 'prog', 7)
+                    end
+                end,
+
+                [44] = function(player, csid, option, npc)
+                    player:tradeComplete()
+                    player:setPos(-116.599, -122.103, -620.01, 253, xi.zone.MINE_SHAFT_2716)
+                    quest:setVar(player, 'prog', 8)
+
+                end,
+            },
+        },
+
+        [xi.zone.LOWER_JEUNO] =
+        {
+            ['Muckvix'] =
+            {
+                onTrade = function(player, npc, trade)
+                    if npcUtil.tradeHasExactly(trade, xi.items.BULB_OF_MISAREAUX_GARLIC) and
+                    quest:getVar(player, 'prog') == 1 then
+                        return quest:progressEvent(99, 0, 0, 0, 0, xi.items.BULB_OF_MISAREAUX_GARLIC)
+                    end
+                end,
+
+                onTrigger = function(player, npc)
+                    if quest:getVar(player, 'prog') == 2 then
+                        return quest:progressEvent(46)
+                    elseif quest:getVar(player, 'prog') == 4 then
+                        return quest:progressEvent(100)
+                    end
+                end,
+            },
+
+            onEventFinish =
+            {
+                [99] = function(player, csid, option, npc)
+                    player:tradeComplete()
+                    npcUtil.giveKeyItem(player, xi.ki.LETTER_FROM_MUCKVIX)
+                    npcUtil.giveCurrency(player, "gil", 2000)
+                    quest:setVar(player, 'prog', 2)
+                end,
+
+                [100] = function(player, csid, option, npc)
+                    player:delKeyItem(xi.ki.PROVIDENCE_POT)
+                    player:delKeyItem(xi.ki.LETTER_FROM_MAGRIFFON)
+                    player:delKeyItem(xi.ki.LETTER_FROM_MUCKVIX)
+                    player:messageSpecial(zones[player:getZoneID()].text.KEYITEM_LOST, xi.ki.PROVIDENCE_POT)
+                    player:messageSpecial(zones[player:getZoneID()].text.KEYITEM_LOST, xi.ki.LETTER_FROM_MAGRIFFON)
+                    player:messageSpecial(zones[player:getZoneID()].text.KEYITEM_LOST, xi.ki.LETTER_FROM_MUCKVIX)
```

Comment:
Just real quick: In the capture I don't remember seeing any text saying I lost these key items. did you? We might not need these lines.

Reply:
I'd have to check retail to confirm but this is based on the description on the wiki of delivering the items, and me asking whether or not you still had your key items at the end of the quest.

Reply:
Just confirmed:  My retail account _does not_ have these items any longer.  Safe to say that the items are indeed removed.